core/src/avm2/globals/flash/crypto.rs

```diff
@@ -16,11 +16,7 @@ pub fn generate_random_bytes<'gc>(
         .unwrap_or(&Value::Undefined)
         .coerce_to_u32(activation)?;
     if !(1..1025).contains(&length) {
-        return Err(Error::AvmError(error(
```

Comment:
`getRandomBytes` throws Error 2004, not ArgumentError 2004

Reply:
Yeah... Noticed that after making this PR :neutral_face: 

I did not expect one error # to have multiple possible types.